src/main/java/Lotto.java

```diff
@@ -4,6 +4,7 @@
 public class Lotto {
 
     private final List<Integer> numbers;
+    private static final int LOTTO_COUNT=6;
```

Comment:
사소한 부분이지만
공백으로 구분해주면 좋을 것 같아요!

IntteliJ를 사용하고있다면
코드 포맷팅 기능을 사용하면 편리합니다 ㅎㅎ
- https://ddiyeon.tistory.com/87

```suggestion
    private static final int LOTTO_COUNT = 6;
```